tests/test_schema.py

```diff
@@ -1853,6 +1860,31 @@ class Model(BaseModel):
     }
 
 
+if OrderedDict:
+
+    def test_typed_ordered_dict():
+        class Model(BaseModel):
+            a: OrderedDict[int, str] = OrderedDictCollections([[1, 'hi']])
+
+        assert Model.schema() == {
+            'title': 'Model',
+            'type': 'object',
+            'properties': {
+                'a': {
+                    'title': 'A',
+                    'default': OrderedDictCollections([[1, 'hi']]),
+                    'type': 'array',
```

Comment:
surely the input type here should be an object? Python will perserve the order of the dict.